app/src/main/java/org/mozilla/focus/menu/browser/BrowserMenuAdapter.kt

```diff
@@ -29,11 +29,11 @@ class BrowserMenuAdapter(
     sealed class MenuItem {
         open val viewType = 0
 
-        open class Default(val id: Int, val label: String) : MenuItem() {
+        open class Default(val id: Int, val label: String, val drawableResId: Int) : MenuItem() {
             override val viewType = MenuItemViewHolder.LAYOUT_ID
         }
 
-        class Custom(id: Int, label: String, val pendingIntent: PendingIntent) : Default(id, label) {
+        class Custom(id: Int, label: String, drawableResId: Int, val pendingIntent: PendingIntent) : Default(id, label, drawableResId) {
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.